<launch>
        <rosparam command="load" file="$(find pc_guided_grasp_planning)/launch/arm5e_pc_grasp_planning.yaml" />

        <arg name="input_basename" default="output" />
	<param name="input_basename" value="$(find laser_stripe_reconstruction)/data/$(arg input_basename)" />
	
	<param name="resources_data_path" value="$(find pc_guided_grasp_planning)/resources" />

        <node name="arm5e_pc_grasp_planning" pkg="pc_guided_grasp_planning" type="arm5e_pc_grasp_planning" output="screen" respawn="false"/>
</launch>


